Rubberduck.VBEEditor/QualifiedSelection.cs

```diff
@@ -12,6 +13,8 @@ public QualifiedSelection(QualifiedModuleName qualifiedName, Selection selection
 
         public QualifiedModuleName QualifiedName { get; }
 
+        public string QualifiedNameAndComponentType => $"{QualifiedName} - {QualifiedName.ComponentType}";
```

Comment:
This does not belong here.

This is clearly a display concern, which does not belong here or on the QMN itself. This should be handled in some converter in the code responsible for displaying the information.